packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -0,0 +1,292 @@
+        datetimeDisplay = null,
+        isModified = null,
+        hasComment = null,
+        isUnverified = null,
+      } = hasDiastolic;
+
+      if (!isEmpty(result.unit)) { compareUnits.diastolic = result.unit.trim().toLowerCase(); }
+      if (!isEmpty(status)) { compareStatusIsInError.diastolic = checkIsStatusInError(hasDiastolic); }
+      if (!isEmpty(conceptDisplay)) { compareConceptDisplays.diastolic = conceptDisplay.trim().toLowerCase(); }
+      if (!isEmpty(datetimeDisplay)) { compareDatetimeDisplays.diastolic = datetimeDisplay.trim().toLowerCase(); }
+      if (!isEmpty(isModified)) { hasModifiedIcon = isModified; }
+      if (!isEmpty(hasComment)) { hasCommentIcon = hasComment; }
+      if (!isEmpty(isUnverified)) { hasUnverifiedIcon = isUnverified; }
+    }
+
+    if (hasSystolic || hasDiastolic) {
+      if ((compareUnits.systolic === compareUnits.diastolic) && !compareStatusIsInError.systolic) {
+        const systolicDisplay = (
+          <Observation
+            key={`Observation-Systolic-${resultData.systolic.eventId}`}
+            eventId={resultData.systolic.eventId}
+            result={resultData.systolic.result}
+            interpretation={!compareStatusIsInError.systolic ? resultData.systolic.interpretation : null}
+            isUnverified={resultData.systolic.isUnverified}
+            hideUnit
+          />
+        );
+        decoratedResultDisplay.push(systolicDisplay);
+      } else {
+        if (!hasSystolic) decoratedResultDisplay.push(<ResultError key={`Error-Systolic-${resultData.id}`} />);
+        if (noDataSystolic) decoratedResultDisplay.push(<NoData key={`NoData-Systolic-${resultData.id}`} />);
+        else if (hasSystolic) {
+          const systolicDisplay = (
+            <ConditionalWrapper
+              key={`del-Systolic-${hasSystolic.eventId}`}
+              condition={compareStatusIsInError.systolic}
+              wrapper={children => <del>{children}</del>}
+            >
+              <Observation
+                key={`Observation-Systolic-${resultData.systolic.eventId}`}
+                eventId={resultData.systolic.eventId}
+                result={resultData.systolic.result}
+                interpretation={!compareStatusIsInError.systolic ? resultData.systolic.interpretation : null}
+                isUnverified={resultData.systolic.isUnverified}
+                hideUnit={hideUnit}
+              />
+            </ConditionalWrapper>
+          );
+          decoratedResultDisplay.push(systolicDisplay);
+        }
+      }
+      decoratedResultDisplay.push(<span key={`Observation-Separator-${(hasSystolic) ? hasSystolic.eventId : hasDiastolic.eventId}`} className={cx('result-display-separator')}>/</span>);
+      if (!hasDiastolic) decoratedResultDisplay.push(<ResultError key={`Error-Diastolic-${resultData.id}`} />);
+      else if (noDataDiastolic) decoratedResultDisplay.push(<NoData key={`NoData-Diastolic-${resultData.id}`} />);
+      else if (hasDiastolic) {
```

Comment:
Lets add braces around if statements and not make them one line for readability

```suggestion
      if (!hasDiastolic) {
          decoratedResultDisplay.push(<ResultError key={`Error-Diastolic-${resultData.id}`} />);
      } else if (noDataDiastolic) {
          decoratedResultDisplay.push(<NoData key={`NoData-Diastolic-${resultData.id}`} />);
      } else if (hasDiastolic) {
```

Also, should this be reduced to an if-else if -else statement?

```jsx
      if (!hasDiastolic) {
          decoratedResultDisplay.push(<ResultError key={`Error-Diastolic-${resultData.id}`} />);
      } else if (noDataDiastolic) {
          decoratedResultDisplay.push(<NoData key={`NoData-Diastolic-${resultData.id}`} />);
      } else {
```